datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/tile/README.md

```diff
@@ -0,0 +1,5 @@
+# Tile-only Integration
+
+Choose this type of integration if you host the logic somewhere you cannot publish  the tile and assets.
```

Comment:
```suggestion
Choose this type of integration if you host the logic externally, and you cannot publish the tile and assets.
```

Reply:
Please confirm this is still accurate

Reply:
Hmm, what I'm trying to say is this:

The logic for the integration lives either in an internal repo (which I don't really want to name specifically) or in the `datadog-agent` repo. The tiles aren't published from those repos, so they are added here.

Let me know how you think I should phrase it best.

Reply:
Thanks for clarifying, I've updated the suggestion.